components/org.wso2.carbon.identity.api.user.association/org.wso2.carbon.identity.api.user.association.v1/src/main/resources/association.yaml

```diff
@@ -624,6 +662,78 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Property'
+
+  BulkFederatedAssociationRequest:
+    type: object
+    properties:
+      failOnErrors:
+        type: integer
```

Comment:
Add a description with the applicable values

Reply:
Addressed with 7b05bfc37c4df1b5a110da7c50a39e73507638c9